053-SentinelAutomatedResponse/Coach/Solution-01.md

```diff
@@ -0,0 +1,30 @@
+# Challenge 01 - Architecture, Agents, Data Connectors and Workbooks - Coach's Guide
+
+**[Home](./README.md)** - [Next Challenge>](./Solution-02.md)
+
+## Introduction
+In this Coach's guide, you fill find guidance, code, and examples that will help you guide the teams over the course of the WTH. In the spirit of continuous improvement, update this repository with any suggestions, altertnatives, or additional challenges.
+
+This section of the hack includes a optional [Intro To Sentinel](Intro2Sentinel.pptx) that features short presentations to introduce key topics associated with each section of this challengechallenge. 
```

Comment:
This is the PPTX in the /Coach folder.  As per comments on the Coach Guide ToC, you should clean up the extra PPTX files and use a raw link here in this context:
"./Intro2Sentinel.pptx?raw=true"

Reply:
The link is broken.

Please remove "Coach" from your path as the page with this link is already in the Coach folder.  Just use:

"./Intro2Sentinel.pptx?raw=true"